dev-java/sbt-bin/sbt-bin-1.8.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
I think we could also easily bump the EAPI to 8.

Reply:
Change it in the commit [6ea9898](https://github.com/gentoo/gentoo/pull/28474/commits/6ea98988f346aa38e62a5c4078283aebb2338d75), thanks. :3